### PR TITLE
IMPORT / REORG : Cleaning useless imports & Reorganising the Code

### PR DESCRIPTION
**Pull request by oualib**

### CODE REVIEW

File: verticapy/core/vdataframe.py [hunk 1]
```diff
@@ -1,10503 +0,0 @@
-"""
```
Comment (mail4umar) [inline verticapy/core/vdataframe.py:1]: So now vdataframe is an empty file?

File: verticapy/datasets/datasets.py [hunk 2]
```diff
@@ -24,15 +24,16 @@
...
 # VerticaPy Modules
 import verticapy, vertica_python
-from verticapy.utils._decorators import (
-    save_verticapy_logs,
-    check_minimum_version,
-)
-from verticapy import vDataFrame
+from verticapy._version import check_minimum_version
+from verticapy._utils._collect import save_verticapy_logs
+from verticapy.core.vdataframe import vDataFrame
```
Comment (mail4umar) [inline verticapy/datasets/datasets.py:29]: at one place I saw you do:
from verticapy.core.vdataframe.vdataframe import vDataFrame
i think better to be consistent

File: verticapy/utils/_decorators.py [hunk 3]
```diff
@@ -1,84 +0,0 @@
-"""
```
Comment (mail4umar) [inline verticapy/utils/_decorators.py:1]: is this an empty file? or deleted file?

**Review by mail4umar — APPROVED** [review]
The code has been very neatly divided into subfolders. 

Though the names of most of the sub-files are self-explanatory, but in the next step we can add some one liners at the top of the code to help users understand what a particular code file is about.

File: verticapy/core/vdataframe.py (hunk 1 above)
Comment (oualib) [reply verticapy/core/vdataframe.py:1]: it was moved to another repo

File: verticapy/utils/_decorators.py (hunk 3 above)
Comment (oualib) [reply verticapy/utils/_decorators.py:1]: deleted

File: verticapy/datasets/datasets.py (hunk 2 above)
Comment (oualib) [reply verticapy/datasets/datasets.py:29]: done

**Pull request merged**